EasyLearn/app/views/player/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= stylesheet_link_tag "player", "data-turbo-track": "reload" %>
+<body>
```

Comment:
It is not necessary to use the body tag, this tag along with the head and html tags are located in the application.html.erb file in the layouts folder

Reply:
Deleted body tag